common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -154,6 +154,35 @@
     </script>
 </%def>
 
+<%def name="require_page(page_name, class_name)">
+    <%doc>
+      Loads Javascript onto your page synchronously.
+      Uses RequireJS in development and a plain script tag in production.
+      Use this form of require_module for all new code.
+    </%doc>
+    <%
+        body = capture(caller.body)
```

Comment:
Are there any specific expectations about the format of `body`? If so maybe stick them into the `<%doc>`.